Misc/Positions_Area Positions 4 External.txt

```diff
@@ -1,8 +1,44 @@
-;Shanwick
-EGGX_FSS:Shanwick Oceanic:131.800:GXO:GXO:EGGX:FSS:-:-::::::
-
-;Gander
-CZQX_FSS:Gander Oceanic:131.700:QXO:QXO:CZQX:FSS::::::::
+;Shanwick Oceanic
```

Comment:
@jackedwards815 would you be able to check this file in particular please?

Reply:
Gander are changing some freqs for CTP. I'm coordinating with them this weekend (when I have time). So bear with me for a few days

Reply:
https://gyazo.com/9f2b36653d108b86f99938f3f39d64ca

Reply:
@hsugden

Reply:
Thanks, amended